packages/tools/addon/templates/components/cs-active-composition-panel.hbs

```diff
@@ -71,35 +71,37 @@
   {{/each}}
 
   {{!-- These are the fields that exist for the template but are not rendered --}}
-  {{#each modelFields as |field|}}
-    {{#let
-      (get permissions (cs-uid field.model))
-      (eq field.model model)
-    as |fieldModelPermissions isPageModelField|}}
-      {{#cs-collapsible-section
-        class=(concat "cs-toolbox-section "
-          (if
-            (cs-has-errors validationErrors field.model (hash name=field.name))
-            "invalid"))
-        title=(cs-card-meta-caption field.model field.options.caption isPageModelField)
-        opened=(eq field.id openedFieldId)
-        open=(action openField field)
-        close=(action openField null)
-      }}
-        <div class="cs-field-editor-section">
-          {{cs-field-editor
-            content=field.model
-            field=field.name
-            enabled=editingEnabled
-            canUpdate=(and
-              fieldModelPermissions.mayUpdateResource
-              (contains field.name fieldModelPermissions.writableFields)
-            )
-            onchange=(action "validate")
-            errors=(get validationErrors field.name)
-          }}
-        </div>
-      {{/cs-collapsible-section}}
-    {{/let}}
-  {{/each}}
+  {{#unless model.hideEditorField}}
```

Comment:
Is `hideEditorField` a good name as it disables **all** fields from being listed automatically?

Reply:
I'm open for suggestions. it doesn't hide all fields, just the ones that either aren't explicitly on the template or that are not editable

Reply:
Hm, maybe I'm mis-understanding what `modelFields` is - I read this as unless `model.hideEditorField` is `true`, iterate over all of the model's fields and show the respective editors?

Reply:
there's another block above this one that renders `renderedFields` on the editor panel. what I see when I inspect `renderedFields` vs. `modelFields`:

on Master Recording card where we have 4 fields:
`renderedFields`: 4 items
`modelFields: []`

on Master Artist Assignments List card:
`renderedFields`: only Artist Assignments field
`modelFields`: the other 7 items (that we don't want to display)

Reply:
Ah sorry, I get it then. Maybe something like `disableAutoEditorFields` or so would be more appropriate then?

Reply:
Works for me, changed it

Reply:
bit of a nitpick but I think `disableAutoEditorField**s**` would be better as it applies to multiple fields, not just one

Reply:
right. I couldn't remember if it was fine to pluralize in the schema since it's not a `has-many` relationship